src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.html

```diff
@@ -9,7 +9,7 @@
               class="bold">Hosts</td>
           <td>{{ hostsCount }}</td>
         </tr>
-        <tr>
+        <tr *ngIf="!isSimpleDeployment">
```

Comment:
let's use templates instead separate if conditions.
```suggestion
        <tr *ngIf="!isSimpleDeployment; else simpleDeploymentTextTpl">
```

```
<ng-template #simpleDeploymentTextTpl>
......
</ng-template>
```